FormlyMapper/FormlyFieldInterface.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace Linio\DynamicFormBundle\FormlyMapper;
+
+interface FormlyFieldInterface
+{
+    /**
+     * @param array $configuration
+     */
+    public function setFieldConfiguration(array $configuration);
+
+    /**
+     * @return array
+     */
+    public function getFormlyFieldConfiguration();
+
+    /**
+     * @return string
+     */
+    public function getTemplateFieldType();
```

Comment:
Is there a reason we need to have Template in the name? Isn't `getFieldType()` simple yet descriptive enough?